### PR TITLE
Google auth

### PR DESCRIPTION
**Pull request by SaloniSS**
# Description

This deals with login with google. I need help with changing it to typescript.

### CODE REVIEW

File: app/components/GoogleAuth.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                })
+                .catch(function (error) {
+                    console.log("Error in adding user", error.response);
+                });
+            }
+            else{
+                console.log("Error in getting user", error);
+            }
+        })
+        .then(function () {});
```
Comment (ReshmiCode) [inline app/components/GoogleAuth.tsx:48]: wots dis empty then for

File: app/components/GoogleAuth.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+              iosStandaloneAppClientId: IOS_APP_CLIENT_ID,
+              scopes: ["profile", "email"],
+            });
+      
+            //If login was successful, create a new profile and then go navigate to MainStack(Bottom Nav)
+            if (LogInResult.type === "success") {
+              //console.log("Logged in", LogInResult.user);
+              newProfile(LogInResult.user);
+              navigate.navigate('MainStack');
+              return LogInResult.accessToken;
```
Comment (ReshmiCode) [inline app/components/GoogleAuth.tsx:67]: why does this function return

File: app/components/GoogleAuth.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            console.log("User already exists");
+        })
+        .catch(function (error) {
+            //If user does not exist, add in database
+            if (error.response.status === 404){
+                console.log("User does not exist");
+                axios
+                .post("https://productivitree.wl.r.appspot.com/api/v1/users", newUser)
+                .then(function (response) {
+                    console.log("User added");
```
Comment (ReshmiCode) [inline app/components/GoogleAuth.tsx:38]: are we adding context in this PR or another

Comment (SaloniSS) [reply app/components/GoogleAuth.tsx:38]: Yes

File: app/components/GoogleAuth.tsx (hunk 1 above)
Comment (SaloniSS) [reply app/components/GoogleAuth.tsx]: Fixed it

File: app/components/GoogleAuth.tsx (hunk 2 above)
Comment (SaloniSS) [reply app/components/GoogleAuth.tsx]: Fixed it

**Pull request merged**